src/core/tsi/ssl_transport_security.cc

```diff
@@ -472,7 +485,7 @@ static tsi_result peer_from_x509(X509* cert, int include_certificate_type,
     }
 
     result = peer_property_from_x509_subject(
-        cert, &peer->properties[current_insert_index++]);
+        cert, &peer->properties[current_insert_index++], false);
```

Comment:
nit: Please add an inline comment here /*is_verified_root_cert=*/ before the "false"

Reply:
Will do